js/data/locale/yo/unitfmt.json

```diff
@@ -8,11 +8,11 @@
             "degree": "#{n}°",
             "arc-minute": "#{n}′",
             "arc-second": "#{n}″",
-            "square-kilometer": "#{n} km²",
-            "hectare": "#{n} ha",
-            "square-meter": "#{n} m²",
-            "square-centimeter": "#{n} cm²",
-            "square-mile": "#{n} mi²",
+            "square-kilometer": "#{n} square-kilometer",
```

Comment:
The symbol is better than the English identifier in these cases. I would put it back to #{n} km²

Reply:
I updated it.